mass_merge/models/record_merge_criteria.py

```diff
@@ -0,0 +1,168 @@
+# Copyright 2019 Digital5 S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+from odoo.tools.safe_eval import safe_eval as eval
+from odoo.exceptions import ValidationError
+
+
+class RecordMergeCriteria(models.Model):
+    _name = 'record.merge.criteria'
+    _description = 'Record Merge by Criteria'
+    _inherit = ['record.merge.mixin']
+
+    domain = fields.Char(string="Filter", required=True)
+    key = fields.Text(string="Group key", required=True,
+                      help="The key that will be used to group the records to merge."
```

Comment:
Identation

Reply:
Hey @Kiplangatdan I don't see any indentation issue here.

Reply:
Indent the same way as the other fields. (But also put in all fields the final `)` on a separate line and put a comma also after the last keyword. Unless a field definition fits all on one line (that is in 88 positions).